content/cloud-docs/api-docs/providers.mdx

```diff
@@ -254,89 +318,1138 @@ curl \
+| `:version`           | The version for which the platform is being deleted.                                                                                                                                                                 |
+| `:os`                | The os of the provider platform that is being deleted.                                                                                                                                                               |
+| `:arch`              | The architecture of the provider platform that is being deleted.                                                                                                                                                     |
+
+[permissions-citation]: #intentionally-unused---keep-for-maintainers
+
+| Status  | Response                  | Reason                                                      |
+| ------- | ------------------------- | ----------------------------------------------------------- |
+| [204][] | Nothing                   | Success                                                     |
+| [403][] | [JSON API error object][] | Forbidden - public provider curation disabled               |
+| [404][] | [JSON API error object][] | Provider not found or user not authorized to perform action |
+
+
+### Sample Request 
+
+```shell
+curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request DELETE \
+  https://app.terraform.io/api/v2/organizations/hashicorp/registry-providers/private/hashicorp/aws/versions/3.1.1/platforms/linux/amd64
+```
+
+## Get a Provider
+
+`GET /organizations/:organization_name/registry-providers/:registry_name/:namespace/:name`
+
+### Parameters
+
+| Parameter            | Description                                                                                                  |
+| -------------------- | ------------------------------------------------------------------------------------------------------------ |
+| `:organization_name` | The name of the organization the provider belongs to.                                                        |
+| `:registry_name`     | Whether this is a publicly maintained provider or private. Must be either `public` or `private`.             |
+| `:namespace`         | The namespace of the provider. For private providers this is the same as the `:organization_name` parameter. |
+| `:name`              | The provider name.                                                                                           |
+
+| Status  | Response                                             | Reason                                                    |
+| ------- | ---------------------------------------------------- | --------------------------------------------------------- |
+| [200][] | [JSON API document][] (`type: "registry-providers"`) | The request was successful                                |
```

Comment:
Again - should we just change all of these to "Success"?

Reply:
yes, so much simpler!